Week1/homework/js-exercises/logRemainder.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
+//1. The answer is 1. The number 6 can be divided by 3 so the remainder after the division is 1.
+//2. The answer is 1. The number 20 can be divided by 4 so the remainder after the division is 1.
+//3. The answer is 1. The number 12 can be divided by 2 so the remainder after the division is 1.
```

Comment:
Correct! Here is also the way you can find every remainder (including huge number):

```
Let have:
5 / 2 = 2.5
Then we say, 2 * 2 = 4
Afterwards, 5 - 4 = 1

So in general is:
a / b = c.d
Then c * b = e,
And a - e = remainder
```